README.md

```diff
@@ -73,15 +75,41 @@ Configuration lives in `~/.nixpkgs/darwin-configuration.nix`. Check out
 There is also preliminary support for building your configuration using a [flake](https://nixos.wiki/wiki/Flakes).  This
 is mostly based on the flake support that was added to NixOS.
 
-A minimal example of using an existing configuration.nix:
+<details>
+<summary>
+
+Getting started from scratch
+
+</summary>
+
+If you don't have an existing `configuration.nix`, you can run the following commands to generate a basic `flake.nix` inside `~/.config/nix-darwin`:
+
+```bash
+mkdir -p ~/.config/nix-darwin
+cd ~/.config/nix-darwin
+nix flake init -t nix-darwin
+```
+
+> NOTE: Make sure to change `system` to `aarch64-darwin` if you are using Apple Silicon.
```

Comment:
This should reference `nixpkgs.hostPlatform` now.

Reply:
Updated :+1: